tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -131,16 +248,39 @@ package <file.genPackage> // <file.grammarName>
 package parser // <file.grammarName>
 <endif>
 
-import "github.com/antlr/antlr4/runtime/Go/antlr"
+// See base listener file for example implementations
```

Comment:
I am not generation a base visitor. The code generated is a template visitor that can be copied uncommented and used as a stating point.